ideascube/serveradmin/catalog.py

```diff
@@ -322,6 +325,26 @@ def remove(self, install_dir):
         for path in glob(os.path.join(datadir, '*', zimname)):
             rm(path)
 
+    # [FIXME] Thoses two properties look like hacks.
+    # We may want to find a way to find those information in the package or
+    # catalog metadata.
+    # For now, use special cases.
+    @property
+    def theme(self):
+        base_name, _ = self.id.rsplit('.', 1)
+        if base_name in ("wikipedia", "wikivoyage", "vikidia"):
+            return "discover"
```

Comment:
The theme is used twice in the kiwix template :
- As the css class to use (so must not be translated)
- As a text. And it is translated in the template.

So no need for me to translate the theme here.